### ISSUE TITLE
Mesh obtained with marching cubes is oversized

### ISSUE THREAD

**Issue by fbottarel** [mikedh/trimesh#922]
Hello @mikedh, first of all thanks for the amazing job. 

I am trying to use `trimesh.voxel.ops.points_to_marching_cubes` to meshify a .pcd point cloud. The algorithm works, but the mesh I get is huge compared to what I started with. 

A snippet of my code is something like

```
pc = trimesh.PointCloud(np.asarray(v.points))
pitch = pc.extents.max() / 10
mcubes = trimesh.voxel.ops.points_to_marching_cubes(pc.vertices, pitch=pitch)
```

```
pc.extents
array([0.10566304, 0.14996433, 0.15259285])

mcubes.extents
array([ 8., 11., 11.])
```

Am I doing something wrong here, or is there a bug somewhere? Thanks, cheers and stay safe

**Comment by mikedh** [comment]
Hmm, that's weird! There was an issue like this a while ago, have you upgraded trimesh recently (`pip install --upgrade trimesh`?). It looks OK in master:

```
import trimesh
import numpy as np

if __name__ == '__main__':
    trimesh.util.attach_to_log()

    p = 10 + np.random.random((100, 3)) * [0.10566304, 0.14996433, 0.15259285]
    pc = trimesh.PointCloud(p)

    pitch = pc.extents.max() / 10
    mcubes = trimesh.voxel.ops.points_to_marching_cubes(pc.vertices, pitch=pitch)
```
Gives something that looks about right:
```
In [1]: pc.extents
Out[1]: array([0.10433, 0.14894, 0.15242])

In [2]: mcubes.extents
DEBUG           caching.py:425   1 items cleared from cache: ['vertex_normals']
DEBUG           caching.py:116   `referenced_vertices`: 5.33E-05s, `cache.verify`: 8.82E-06s
DEBUG           caching.py:116   `bounds`: 5.76E-04s, `cache.verify`: 1.26E-05s
DEBUG           caching.py:116   `extents`: 7.49E-04s, `cache.verify`: 2.86E-04s
Out[2]: array([0.12194, 0.16766, 0.16766])
```

**Comment by fbottarel** [comment]
Thanks @mikedh. Apparently I was using an old virtual environment which had trimesh 3.6.30 installed. I upgraded to the latest master version and the issue has disappeared. 

Closing this, thanks again!

**Comment by mikedh** [comment]
No worries, yeah looks like it was fixed pretty recently in [3.7.12](https://github.com/mikedh/trimesh/commit/5abd7553727ab9294d6b9bf475e11c33784ae391).
